bindings/go/descriptor/local_access.go

```diff
@@ -0,0 +1,33 @@
+package descriptor
+
+import (
+	"ocm.software/open-component-model/bindings/go/runtime"
+)
+
+// LocalBlobAccessType is the access type of blob local to a component.
+const (
+	LocalBlobAccessType   = "localBlob"
+	LocalBlobAccessTypeV1 = LocalBlobAccessType + "/" + "v1"
+)
+
+// LocalBlob describes the access for a local blob.
+type LocalBlob struct {
+	runtime.Type `json:"type"`
+	// LocalReference is the repository local identity of the blob.
+	// it is used by the repository implementation to get access
+	// to the blob and if therefore specific to a dedicated repository type.
+	LocalReference string `json:"localReference"`
+	// MediaType is the media type of the object represented by the blob
+	MediaType string `json:"mediaType"`
+	// GlobalAccess is an optional field describing a possibility
+	// for a global access. If given, it MUST describe a global access method.
+	GlobalAccess *runtime.Raw `json:"globalAccess,omitempty"`
```

Comment:
Still on a pointer (maybe needed? unsure)

Reply:
It kind-a has to be for a propert omit empty.. 🤔 Otherwise the checks are going to be crazy not to mention the resulting JSON will have some empty values instead of values completely not being their. Since Raw is has an embedded type field.